iothub/device/src/Transport/AmqpIot/AmqpIotErrorAdapter.cs

```diff
@@ -26,7 +26,6 @@ internal static class AmqpIotErrorAdapter
         public static readonly AmqpSymbol ArgumentError = AmqpIotConstants.Vendor + ":argument-error";
         public static readonly AmqpSymbol ArgumentOutOfRangeError = AmqpIotConstants.Vendor + ":argument-out-of-range";
         public static readonly AmqpSymbol DeviceContainerThrottled = AmqpIotConstants.Vendor + ":device-container-throttled";
-        public static readonly AmqpSymbol PartitionNotFound = AmqpIotConstants.Vendor + ":partition-not-found";
```

Comment:
the service team confirmed that this error code should never be returned to clients.